customTests/custom-test-setup.sh

```diff
@@ -40,7 +40,7 @@ function install_perf_test(){
 	fi
 
 	pushd ${EXE_DIR}
-	perftest_dir="perftest-${VERSION}"
+	perftest_dir="perftest-4.5"
```

Comment:
Is it possible to grab 1st field before "-" of `VERSION`? Hardcoded version is used here while line 45 still uses variable

Reply:
let me check

Reply:
fixed